src/components/utils/Input/InputElement.tsx

```diff
@@ -24,7 +24,6 @@ const InputElement = React.forwardRef<HTMLInputElement | HTMLTextAreaElement, In
       <Box
         ref={ref}
         as="input"
-        verticalAlign="top"
```

Comment:
I believe the reason you had to remove this is because your prefix is a `span`. If it was a `div` or anything else you wouldn't need to.

I'll have to verify that this hasn't caused problems in other places

Reply:
The cleanup was due to Firefox,  validating the input. `verticalAlign` applies to `inline-block` elements and the `input` elements are just `inline`.

Reply:
Gotcha. Ok, resolving this one and will re-open if I find that something looks misaligned :)